R/rustfft/build_tarballs.jl

```diff
@@ -8,8 +8,8 @@ uuid = Base.UUID("a83860b7-747b-57cf-bf1f-3e79990d037f")
 delete!(Pkg.Types.get_last_stdlibs(v"1.6.3"), uuid)
 
 name = "rustfft"
-version = v"0.3.0"
-julia_versions = [v"1.6.3", v"1.7", v"1.8", v"1.9"]
+version = v"0.3.1"
```

Comment:
To be clear: you only want to build for Julia v1.10 without changing the upstream source?